### PR TITLE
feat: environment variable support for flags

### PR DESCRIPTION
**Pull request by imhugofonseca**
#161

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,16 +83,16 @@ Here's the available flags:
...
+| `--port`(int)                   | PORT                | port number to use for web ui server                                | 8080             |
+| `---redis-url`(string)          | REDIS_URL           | URL to redis server                                                 | ""               |
+| `--redis-addr`(string)          | REDIS_ADDR          | address of redis server to connect to                               | "127.0.0.1:6379" |
+| `--redis-db`(int)               | REDIS_DB            | redis database number                                               | 0                |
+| `--redis-password`(string)      | REDIS_PASSWORD      | password to use when connecting to redis server                     | ""               |
+| `--redis-cluster-nodes`(string) | REDIS_CLUSTER_NODES | comma separated list of host:port addresses of cluster nodes        | ""               |
+| `--redis-tls`(string)           | REDIS_TLS           | server name for TLS validation used when connecting to redis server | ""               |
+| `--redis-insecure-tls`(bool)    | REDIS_INSECURE_TLS  | disable TLS certificate host checks                                 | false            |
```
Comment (hibiken) [inline README.md:95]: Just tested locally but I don't think these are working, because the flag uses "-" (dash) instead of "_" (underscore).

**Review by hibiken — COMMENTED** [review]
Thank you so much for working on this. 

If we add a new dep, I think go.mod and go.sum files should be updated?
But since we are using dashes for flag names, seems like the package (github.com/namsral/flag) doesn't quite work for us. 

Since it's not that complicated to fallback to Env variables if flag is not set, I think we should implement ourselves without relying on a 3P package.

**Comment by imhugofonseca** [comment]
It works well even with dashes it just replaces them with underscores. But I understand not wanting another package installed. I will add this locally

**Comment by imhugofonseca** [comment]
@hibiken Hey! I've integrated this locally! I can create tests if it can wait :)

**Comment by hibiken** [comment]
LGTM! thanks for working on this 🙏

**Pull request merged**